lms/static/scripts/frontend_apps/components/dashboard/test/DashboardActivityFilters-test.js

```diff
@@ -484,6 +495,161 @@ describe('DashboardActivityFilters', () => {
     });
   });
 
+  context('when segments are provided', () => {
+    let onSegmentsChange;
+
+    beforeEach(() => {
+      onSegmentsChange = sinon.stub();
+    });
+
+    function createComponentWithSegments(segments = {}) {
```

Comment:
```suggestion
    function createComponentWithSegments(segmentsConfig = {}) {
```

The name sounds like it should be a list of segments, but rather it refers to an object containing segments-related properties.